eng/Versions.props

```diff
@@ -38,6 +38,9 @@
     <MicrosoftVSSDKBuildToolsVersion>17.9.3137-preview3</MicrosoftVSSDKBuildToolsVersion>
     <vswhereVersion>2.4.1</vswhereVersion>
     <MicrosoftDevDivOptimizationDataPowerShellVersion>1.0.803</MicrosoftDevDivOptimizationDataPowerShellVersion>
+
+    <!-- TODO: remove https://github.com/dotnet/roslyn/issues/71827 -->
+    <MicrosoftDotNetXliffTasksVersion>9.0.0-beta.24076.5</MicrosoftDotNetXliffTasksVersion>
```

Comment:
@rainersigwald The linked issue #71827 relates to needing to upgrade to Arcade 9. Is there any easy way here to have MSBuild report a build warning on this line if Arcade 9 is used (thus forcing a correction here once we update)?

Reply:
Not easy in this spot--you'd have to have a target conditionally emit an error.